todo/tests/test_utils.py

```diff
@@ -56,24 +56,24 @@ def test_send_email_to_thread_participants(todo_setup, django_user_model, email_
 
 
 def test_defaults(settings):
-    """todo's `defaults` module provides reasonable default values for unspecified settings.
-    If a value is NOT set, it should be pulled from the hash in defaults.py.
+    """todo's `settings` module provides reasonable default values for unspecified settings.
+    If a value is NOT set, it should be pulled from the defaults in settings.py.
     If a value IS set, it should be respected.
-    n.b. TODO_STAFF_ONLY which defaults to True in the `defaults` module."""
+    n.b. TODO_STAFF_ONLY which defaults to True in the `settings` module."""
```

Comment:
I'm not understanding the point of this change.

Reply:
Explained above.